__mocks__/adobexd/sample_dump.json

```diff
@@ -0,0 +1 @@
+undefined
```

Comment:
If you regenerate this file it should update with the response provided by Adobe XD parser